include/deal.II/lac/swappable_vector.h

```diff
@@ -128,11 +128,14 @@ class SwappableVector : public Vector<number>
   /**
    * Remove the file to which the data has been stored the last time. After
    * this, the object does not own any file any more, so of course you can't
-   * call @p reload no more.
+   * call @p reload again.
```

Comment:
Ah, from the time when I thought that using slang in documentation was a good idea :-)